CHANGELOG.md

```diff
@@ -2,15 +2,31 @@
 
 ### NEXT
 
-* Fixes bare variables in conditionals (deprecation warnings with Ansible 2.8+) (#204)
+#### New features
+
+* Report when Ruby is removed and don't ignore when version listing fails (#212)
+
+#### Bug fixes
+
+* Fix bare variables in conditionals (deprecation warnings with Ansible 2.8+) (#204)
 * Fix boolean values when checking array (#207)
-* Fixes regression forbidding installation of rubies (#209, fixed via #207)
+* Fix regression forbidding installation of rubies (#209, fixed via #207)
 * Fix conditional logic for detecting and removing versions of Ruby (#212)
-* Report when Ruby is removed and don't ignore when version listing fails (#212)
+
+#### Changes
+
+* Extend test matrix with: (#215)
+  * CentOS 8, Debian 10, Ubuntu 18.04 and 20.04
+  * Ruby 2.6.6 and 2.7.1
+* Modify some test assertions to better abstract RVM behaviour from its internal implementation ("blackbox" testing) (#205)
+* Fix and enhance Travis CI setup (#205)
+  * Install more recent versions of Ruby, to be compatible with latest Bundler version
+  * Validate system-wide setup (root.yml)
+  * Parallelize the tests to speed up the builds
```

Comment:
@pkuczynski awesome, thank you very much. I like the new "Categorization" grouping.